### PR TITLE
Fix type checking for subclasses of types

### PR DESCRIPTION
**Pull request by jrc-exp**
When restoring the pre-trained EfficientNet B4 weights in the example notebook, the code fails to replace BatchNorm2D with GhostNorm because the timm repo uses a subclass of nn.BatchNorm2d which fails the type check in the current code: 

https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/layers/norm_act.py#L26

"isinstance" is a safer way to do the type checking.

### CODE REVIEW

**Comment by jrc-exp** [comment]
Looks like updates pushed after I started this fix this issue.
